localStorage.js

```diff
@@ -0,0 +1,23 @@
+const emailStore = document.querySelector('#email');
+const nameStore = document.querySelector('#contact');
+const formListen = document.querySelector('form');
+
+const storeData = {
+  name: '',
+  email: '',
+};
```

Comment:
- [ ] Include the text input value from the textarea field as well. It's value needs to be stored in the localStorage